snekbox/memfs.py

```diff
@@ -135,12 +138,17 @@ def files(
             exclude_files: A dict of Paths and last modified times.
                 Files will be excluded if their last modified time
                 is equal to the provided value.
+            timeout: The maximum time for the file parsing. If exceeded,
+                a TimeoutError will be raised.
```

Comment:
```suggestion
            timeout: The maximum time for the file parsing.
```
The omitted part should instead be in a `Raises:` section.

Reply:
Changed to `Raises:` in [6441719](https://github.com/python-discord/snekbox/pull/173/commits/64417197b58e3945f0b857c7fa648dcbe28254c4)